src/pages/ThorChainLP/components/LpType.tsx

```diff
@@ -115,66 +104,24 @@ export const LpType = ({ assetId, defaultOpportunityId, onAsymSideChange }: Depo
     [assetId],
   )
 
-  const walletSupportsRune = useMemo(() => {
-    const _walletSupportsRune = walletSupportsChain({
-      chainId: thorchainChainId,
-      wallet,
-      isSnapInstalled,
-    })
-
-    return _walletSupportsRune && thorchainAccountIds.length > 0
-  }, [isSnapInstalled, thorchainAccountIds.length, wallet])
-
-  const walletSsupportsAsset = useMemo(() => {
-    const _walletSupportsAsset = walletSupportsChain({
-      chainId: fromAssetId(assetId).chainId,
-      wallet,
-      isSnapInstalled,
-    })
-
-    return _walletSupportsAsset && poolAssetAccountIds.length > 0
-  }, [assetId, isSnapInstalled, poolAssetAccountIds.length, wallet])
-
-  const defaultValue = useMemo(() => {
-    if (walletSupportsRune && walletSsupportsAsset) return 'sym'
-    if (walletSsupportsAsset) return AsymSide.Asset
-    if (walletSupportsRune) return AsymSide.Rune
-  }, [walletSsupportsAsset, walletSupportsRune])
-
   const { getRootProps, getRadioProps, setValue } = useRadioGroup({
     name: 'depositType',
-    defaultValue,
+    defaultValue: opportunityId ? fromOpportunityId(opportunityId).type : 'sym',
     onChange: onAsymSideChange,
   })
 
   useEffect(() => {
-    if (!defaultValue) {
-      // We've switched to an asset for which none of the a/sym types are supported, make sure the previously selected value is cleared
-      setValue('')
-      return
-    }
-
-    // Reset the radio state to default pool type on assetId change, meaning pool change
-    // This is to ensure the radio is synchronized with the actual default sym pool being selected on pool change
-    setValue(defaultValue)
-  }, [assetId, defaultValue, setValue])
```

Comment:
👀 will need to test this to ensure this is still happy

Reply:
Although reading `Remove asset support checks from LpType to allow user to get quotes on any asset pairs/lp types` the intent seems to be to remove this - so the testing will be about ensuring that continuing is still disabled

Reply:
Yes, I could have clarified more here. Any asset and lp type can be selected, but we will still show the warnings and not allow user to proceed to the next step without necessary wallet support